pallets/loans/src/lib.rs

```diff
@@ -193,7 +193,7 @@ pub mod pallet {
 			+ MaxEncodedLen;
 
 		/// Defines the balance type used for math computations
-		type Balance: tokens::Balance + FixedPointOperand;
+		type Balance: tokens::Balance + FixedPointOperand + From<u128>;
```

Comment:
Why is this bound?

Reply:
Needed it to get it to compile but probably there's a better way to do it that I didn't find yet ;)